grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -72,25 +73,25 @@ class ScoreL2RPN2023(ScoreL2RPN2020):
     """
 
     def __init__(
-        self,
-        env,
-        env_seeds=None,
-        agent_seeds=None,
-        nb_scenario=16,
-        min_losses_ratio=0.8,
-        verbose=0,
-        max_step=-1,
-        nb_process_stats=1,
-        scale_assistant_score=100.0,
-        scale_nres_score=100.,
-        weight_op_score=0.6,
-        weight_assistant_score=0.25,
-        weight_nres_score=0.15,
-        weight_confidence_assistant_score=0.7,
-        min_nres_score=-100,
-        add_nb_highres_sim=False,
+            self,
+            env,
+            env_seeds=None,
+            agent_seeds=None,
+            nb_scenario=16,
+            min_losses_ratio=0.8,
+            verbose=0,
+            max_step=-1,
+            nb_process_stats=1,
+            scale_assistant_score=100.0,
+            scale_nres_score=100.,
+            weight_op_score=0.6,
+            weight_assistant_score=0.25,
+            weight_nres_score=0.15,
+            # weight_confidence_assistant_score=0.7,
+            min_nres_score=-100,
+            min_assistant_cost_score=-100,
+            add_nb_highres_sim=False,
```

Comment:
this argument is not at the right place. It should be right after `score_names` to follow the order of ScoreL2RPN2020

Reply:
You mean add_nb_highres_sim ?
In that case it will look something like that( after nb_process_stats actually)
![image](https://github.com/rte-france/Grid2Op/assets/5496312/2ca726ae-bd1f-4ace-b31c-ffd1e0903164)

Do you want me to add scores_func and score_names argument also ? (They are missing in ScoreL2RPN2023, only hardcoded after

Reply:
The base class has constructor:

```python

    def __init__(
        self,
        env,
        env_seeds=None,
        agent_seeds=None,
        nb_scenario=16,
        min_losses_ratio=0.8,
        verbose=0,
        max_step=-1,
        nb_process_stats=1,
        scores_func=L2RPNSandBoxScore,
        score_names=None,
        add_nb_highres_sim=False,
    ):
   ```
   
   To avoid possible issues later on, in the derived class you should keep the same order. 

(also to avoid making print screen then copy paste the image...  you can use:

\`\`\`python 
python code here
 \`\`\`

It's easier and more efficient and other people can copy paste)

Reply:
Done